### DISCUSSION TITLE
Mesh File  Not Able to Use Interface Functions

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#17992] [Q&A Modules: General] [answered]
I have modified an fsi model that incorporates the use of a mes file instead of a mesh block definition. When I try to execute the model I get an error that states SideSetsBetweenSubdomainsGenerator, and SubdomainBoundingBoxGenerator wil not work with a mesh file. Is there an equivalent to these interface functions that will work with a mesh file?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Use a FileMeshGenerator to load your mesh file, then it should work. 

Guillaume

**Reply by TLWise** [reply]
Thank you very much!
